lang/src/org/partiql/lang/errors/ErrorCode.kt

```diff
@@ -649,6 +649,11 @@ enum class ErrorCode(private val category: ErrorCategory,
             "LIMIT value must be an integer but found ${errorContext.getProperty(Property.ACTUAL_TYPE)}}"
     },
 
+    EVALUATOR_NON_TEXT_STRUCT_FIELD (
+        ErrorCategory.EVALUATOR,
+        LOCATION,
+        "Struct field should be text."),
```

Comment:
Is it more accurate to say struct field *names*/*keys* rather than struct field? A struct field has a name/key (restricted to text) and a field (could be any value).

Reply:
Also, I don't think the `messagePrefix` argument of `ErrorCode` should be used. It can lead to a misconfigured error message:

```
PartiQL> { 1 : 2 };
org.partiql.lang.eval.EvaluationException: Found struct field to be of type INT
	Evaluator Error: at line 1, column 3: Struct field should be text., <UNKNOWN> : <UNKNOWN>

	at org.partiql.lang.eval.ExceptionsKt.err(Exceptions.kt:51)
        ...
```

Overriding `getErrorMessage` and leaving `messagePrefix` as an empty string should get rid of the extra `, <UNKNOWN> : <UNKNOWN>` at the end.